sormas-api/src/main/java/de/symeda/sormas/api/vaccination/VaccinationFacade.java

```diff
@@ -35,24 +36,21 @@
 import de.symeda.sormas.api.utils.SortProperty;
 
 @Remote
-public interface VaccinationFacade {
-
-	VaccinationDto save(@Valid VaccinationDto dto);
+public interface VaccinationFacade extends BaseFacade<VaccinationDto, VaccinationIndexDto, VaccinationReferenceDto, VaccinationCriteria> {
 
 	/**
 	 * Creates the passed DTO as an entity and automatically assigns an existing immunization or, if none is found,
 	 * creates a new immunization with the passed region, district, person, and disease, and assigns it to the vaccination.
 	 * Throws an exception when called with a DTO that already has an immunization assigned or has a UUID that already
 	 * exists in the database.
-	 * 
 	 * If the vaccination date is empty, assigns the vaccination to the latest immunization (defined by
 	 * ImmunizationEntityHelper.getDateForComparison).
 	 * If the vaccination date is not empty, assigns an existing immunization according to the following priorities:
 	 * 1. Immunization with start date <= vaccination date <= end date
 	 * 2. Immunization with the nearest start or end date to the vaccination date
 	 * 3. Immunization with the nearest report date to the vaccination date
 	 */
-	VaccinationDto createWithImmunization(
+	VaccinationDto saveWithImmunization(
```

Comment:
Please revert this change; method is named like this for a reason because it's only supposed to be used to CREATE a new vaccination. There's even an exception thrown when trying to call it with an existing vaccination, so calling it "saveWithImmunization" would be misleading